### DISCUSSION TITLE
Is there a "Latest Version" download link?

### DISCUSSION THREAD

**Discussion by droyad** [datalust/seq-tickets#1531] [Help] [answered]
Is there a permalink to the latest seq installer? I want to automate the install/upgrade.

I could use chocolatey but that seems a version behind.

**Comment by nblumhardt** [comment, marked as answer]
Hi Rob!

Unfortunately, you'll need to follow an HTTP redirect, but `https://datalust.co/download/begin?version=latest` will do it.

There's also `https://datalust.co/api/releases/latest`, which will give you a JSON response like:

```json
{"latestRelease":"2022.1.7378"}
```

if you want to plug a specific version into the download URL, e.g. `https://datalust.co/download/begin?version=2022.1.7378`.

Avoid taking a dependency on the structure of the redirect target URL, we're in the process of moving release artifacts from one blob store to another :-)

Hope this helps,
Nick

**Reply by droyad** [reply]
🧑‍🍳💋
```
if(!(Test-Path "C:\temp")) {
    mkdir c:\temp
}

Write-Host "Downloading"

(New-Object System.Net.WebClient).DownloadFile("https://datalust.co/download/begin?version=latest","c:\temp\seq.msi")

Write-Host "Installing"

Start-Process c:\temp\seq.msi -Wait -ArgumentList "/quiet"
Write-Host "Seq is installed"

$version = (. 'C:\Program Files\Seq\Seq.exe' version)
Write-Highlight "Seq is version $version"

netsh http add urlacl url="https://#{Hostname}:443/" user="ops\\SeqDefaultInstance"

. 'C:\Program Files\Seq\Seq.exe' --port 443 --thumbprint "#{Certificate.Thumbprint}" --hostname "#{Hostname}"
. 'C:\Program Files\Seq\Seq.exe' service stop
. 'C:\Program Files\Seq\Seq.exe' service start
```
